website/content/blog/2023-11-getting-started-as-a-tag-env-contributor.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Getting started as a TAG Environmental Sustainability Contributor
+description: Getting started as a TAG Environmental Sustainability Contributor
+slug: tbd
+weight: 1
+date:   2023-11-07 00:00:00 +0000
+author: Antonio Di Turi, Atreay Kukanur, Mfon Nta
+reviewers: Kristina Devochko, Leonard Pahlke, Marlow Weston
+---
+
+## Getting started as a TAG Environmental Sustainability Contributor
+
+<p class="mt-5 mb-5"><img src="/images/blogs/2023-11-getting-started-as-a-tag-environmental-sustainability-contributor/tag-env-logo.png" alt="TAG Environmental Sustainability Logo"></p>
+
+If you are reading this blog it might be that you are looking forward to starting contributing to sustainability in the cloud native world.  This document will help you, and you can start by becoming a member of the TAG or one of its associated working groups.
```

Comment:
```suggestion
If you are reading this blog it might be that you are interested in contributing to the cloud native sustainability domain. This document will help you and guide you through the different possibilities that are available for you, like becoming a member of the TAG Environmental Sustainability or one of its associated working groups.
```

Reply:
> This document will help you

In this blog [...]

Reply:
> becoming a member of the TAG

we do not really have "members" of the tag. Lets rephrase it. There are leadership roles but no membership roles. We do not have membership roles because it is not associated with anything, like specific tasks. So everyone can be considered a member or not. Managing members without any benefit just creates overhead.

Reply:
solved, check last commit